### ISSUE TITLE
Default config location does not work on mac

### ISSUE THREAD

**Issue by raine** [h3poteto/kube-job#67]
```
% kube-job -v run --template-file=test.yaml  --args="echo fuga" --container="test"
INFO[0000] Using config file: $HOME/.kube/config
FATA[0000] stat $HOME/.kube/config: no such file or directory
% ls -l $HOME/.kube/config
-rw-------@ 1 raine  staff  5453 Jul 20 01:34 /Users/raine/.kube/config
```

You have to pass the same config as argument, then it works.

**Comment by h3poteto** [comment]
I can reproduce this problem.
```
$ kube-job run -v --template-file=test.yaml  --args="echo fuga" --container="test" 
INFO[0000] Using config file: $HOME/.kube/config
FATA[0000] stat $HOME/.kube/config: no such file or directory
exit status 1
```

**Comment by h3poteto** [comment]
I fixed this issue, and I released v0.4.3. Please use it.

**Comment by raine** [comment]
```
% kube-job -v run --template-file=job.yaml --container=bikeshopsearch-job --args="ls"
INFO[0000] Using config file: $HOME/.kube/config
INFO[0000] Received args:
INFO[0000] ls
INFO[0000] Starting job: bikeshopsearch-job-1b95e9256232c434864f6087fcd6947f
INFO[0000] Waiting for running job...
backend
node_modules
INFO[0006] Job is succeeded
INFO[0016] Removing the job: bikeshopsearch-job-1b95e9256232c434864f6087fcd6947f
INFO[0016] Remove related pods which labels is: job-name=bikeshopsearch-job-1b95e9256232c434864f6087fcd6947f
```

Works! ありがとうございます。
